scope3ai/tracers/huggingface/text_to_speech.py

```diff
@@ -23,30 +25,68 @@ class TextToSpeechOutput(_TextToSpeechOutput):
 def huggingface_text_to_speech_wrapper_non_stream(
     wrapped: Callable, instance: InferenceClient, args: Any, kwargs: Any
 ) -> TextToSpeechOutput:
-    timer_start = time.perf_counter()
-    response = wrapped(*args, **kwargs)
-    request_latency = (time.perf_counter() - timer_start) * 1000
+    http_response: Response | None = None
+    with requests_response_capture() as responses:
+        response = wrapped(*args, **kwargs)
+        http_responses = responses.get()
+        if len(http_responses) > 0:
+            http_response = http_responses[-1]
+
+    model = kwargs.get("model") or instance.get_recommended_model("text-to-speech")
+    compute_time = http_response.headers.get("x-compute-time")
+    input_tokens = http_response.headers.get("x-compute-characters")
```

Comment:
This will break if http_response is None for any reason (wrapper does not work or such)